Build/test.ps1

```diff
@@ -38,10 +38,29 @@ function Validate-Integrals {
     }
 }
 
+function Test-AutoSubstitution {
```

Comment:
Since it's not generally a scalable approach to have a different cmdlet for each sample being tested, it'd be helpful to explain in comments why autosubstitution is unusual in that regard. Hopefully, that will reduce the temptation to make one-off cmdlets for other samples in the future.